docker/redis.persistent.conf

```diff
@@ -0,0 +1,17 @@
+# Redis configuration for Lisk Service.
+#
+# This config enables persistence
+bind 0.0.0.0
+port 6379
+tcp-backlog 127
+timeout 0
+tcp-keepalive 300
+
+# Snapshotting
+appendonly yes
+aof-use-rdb-preamble yes
+save ""
+
+# Memory management
+maxmemory 512mb
+maxmemory-policy volatile-lru
```

Comment:
For this policy to be effective, won't we also need to ensure that all the DB entries have the expiry (TTL) set?

Ref: https://redis.io/topics/lru-cache
```
volatile-lru: evict keys by trying to remove the less recently used (LRU) keys first, but only among keys that have an expire set, in order to make space for the new data added.
```

Reply:
Yes, only keys with expiry time will be removed, other keys will be persistent.